commands/version.go

```diff
@@ -18,16 +18,15 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
-
+	"github.com/ystia/yorc/v4/config"
 	"github.com/ystia/yorc/v4/helper/consulutil"
 )
 
-var version = "unknown phantom version"
-
 var gitCommit = "dev version"
```

Comment:
I would keep the `version` and `gitCommit` in the same place they describe the same thing.

Reply:
`gitCommit`should somehow be part of the version information  retrieved by the API